app/_components/commodities/Form.tsx

```diff
@@ -153,73 +158,93 @@ const Form: React.FC<FormProps> = ({
-            <ButtonGroup
-              isAttached
-              borderColor={GetColor('border')}
-              variant="outline"
+            <FormControl
+              width="100%"
+              isInvalid={!!errors.minSupply || !!errors.minDemand}
             >
-              <Button
-                variant={isBuying ? 'outline' : 'colorless'}
-                onClick={() => setIsBuying(true)}
+              <FormLabel>I am looking to:</FormLabel>
+              <ButtonGroup
+                isAttached
+                borderColor={GetColor('border')}
+                variant="outline"
               >
-                Buy
-              </Button>
-              <Button
-                variant={isBuying ? 'colorless' : 'outline'}
-                onClick={() => setIsBuying(false)}
-              >
-                Sell
-              </Button>
-            </ButtonGroup>
-          </FormControl>
-          <FormControl width="100%">
-            {isBuying && numberInputs('Minimum Supply', 'minSupply')}
-            {!isBuying && numberInputs('Minimum Demand', 'minDemand')}
-            <FormErrorMessage>
-              {errors.minSupply &&
-                isBuying &&
-                (errors.minSupply.message as string)}
-              {errors.minDemand &&
-                !isBuying &&
-                (errors.minDemand.message as string)}
-            </FormErrorMessage>
-          </FormControl>
-        </Stack>
+                <Button
+                  variant={isBuying ? 'outline' : 'colorless'}
+                  onClick={() => setIsBuying(true)}
+                >
+                  Buy
+                </Button>
+                <Button
+                  variant={isBuying ? 'colorless' : 'outline'}
+                  onClick={() => setIsBuying(false)}
+                >
+                  Sell
+                </Button>
+              </ButtonGroup>
+            </FormControl>
+            <FormControl width="100%">
+              {isBuying && numberInputs('Minimum Supply', 'minSupply')}
+              {!isBuying && numberInputs('Minimum Demand', 'minDemand')}
+              <FormErrorMessage>
+                {errors.minSupply &&
+                  isBuying &&
+                  (errors.minSupply.message as string)}
+                {errors.minDemand &&
+                  !isBuying &&
+                  (errors.minDemand.message as string)}
+              </FormErrorMessage>
+            </FormControl>
+          </Stack>
+        </Collapse>
       </Flex>
-      <Button type="submit" variant="submit" id="submit" isLoading={isLoading}>
-        Submit
-      </Button>
+      <HStack justifyContent="space-between" paddingRight={[0, '40%', '50%']}>
+        <Button
+          type="submit"
+          variant="submit"
+          id="submit"
+          isLoading={isLoading}
+        >
+          Submit
+        </Button>
+        <ExpandIcon isExpanded={isExpanded} setIsExpanded={setIsExpanded} />
```

Comment:
I'm not sure this is obvious enough. Maybe look at adding something like "show/hide advanced filters/options" alongside or above the icon?

Inara uses a massive grey button which is overkill imo, but it does a good job of highlighting there's more options behind a click

![image](https://github.com/ed-pilots-network/frontend/assets/319883/5c80c883-23a1-4cc2-b6ac-5438516c2906)